tests/sqlparser_duckdb.rs

```diff
@@ -508,3 +508,31 @@ fn test_duckdb_named_argument_function_with_assignment_operator() {
         expr_from_projection(only(&select.projection))
     );
 }
+
+#[test]
+fn test_select_wildcard_with_replace() {
```

Comment:
Would it make sense to move the test to `sqlparser_common` so that all dialects can share? (moving the ones in bigquery and clickhouse) - since they all test the same functionality, it would let us not duplicate the same test for each dialect as we do currently in this PR

Reply:
Thanks for the review @iffyio! I moved the tests to `sqlparser_common`